backend/modules/auth/router.py

```
@@ -202,32 +205,125 @@ async def logout_user(response: Response):
+            token=token_value,
+            expires_at=expires_at,
+        )
+        db.add(reset_token)
+        db.commit()
+
+        frontend_url = (settings.frontend_url or FRONTEND_URL).rstrip("/")
+        reset_link = f"{frontend_url}/reset-password?token={token_value}"
+
+        try:
+            await send_password_reset_email(user.email, reset_link)
+        except Exception as exc:  # pragma: no cover — defensive, email service already logs
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)